src/qforte/ucc/uccnvqe.py

```diff
@@ -259,10 +256,17 @@ def get_num_ham_measurements(self):
     # TODO: depricate this function
     def get_num_commut_measurements(self):
         # if self._use_analytic_grad:
-        #     self._n_commut_measurements = self._final_result.njev * (len(self._pool_obj))
+        #     self._n_commut_measurements = self._final_result.njev * (len(self._qubit_pool))
         #     return self._n_commut_measurements
         # else:
         #     return 0
         return 0
 
+    def fill_excited_dets(self):
+        reference_state = qforte.QubitBasis(self._nqb)
+        for k, occ in enumerate(self._ref):
+            reference_state.set_bit(k, occ)
+
+        self._excited_dets = [operator_to_determinant(qubit_operator, reference_state) for _, qubit_operator in self._qubit_pool]
+
```

Comment:
This function is also defined in uccnpqe.py. Maybe place it in utils to avoid the repetition.